akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -1007,14 +1007,14 @@ object GraphDSL extends GraphApply {
     private class PortOpsImpl[+Out](override val outlet: Outlet[Out @uncheckedVariance], b: Builder[_])
       extends PortOps[Out] {
 
-      override def withAttributes(attr: Attributes): Repr[Out] =
-        throw new UnsupportedOperationException("Cannot set attributes on chained ops from a junction output port")
+      override def withAttributes(attr: Attributes): Repr[Out] = throw settingAttrNotSupported
```

Comment:
Wanted the throw line in the right place, but I guess it will anyways be in the stack trace.